Tests/GitBuddyTests/TestHelpers/ReleaseNotesJSON.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  ReleaseNotesJSON.swift
+//  
+//
+//  Created by Antoine van der Lee on 13/06/2023.
+//
+
+import Foundation
+
+// swiftlint:disable line_length
+let ReleaseNotesJSON = """
+
+{
+  "name": "Release v1.0.0 is now available!",
+  "body": "##Changes in Release v1.0.0 ... ##Contributors @monalisa"
+}
+"""
```

Comment:
We could also expose this as a `UTF8View` like so
```suggestion
""".utf8
```

Reply:
... and then call `Data(releasesNotesJSON)` without needing to unwrap it

Reply:
Good idea! I took it a little different, but using your suggestion I removed the force unwraps 👌